common/ip-packet-requests/src/v7/request.rs

```diff
@@ -298,6 +350,29 @@ pub struct SignedDisconnectRequest {
     pub signature: Option<Vec<u8>>,
 }
 
+impl SignedRequest for SignedDisconnectRequest {
+    fn identity(&self) -> &identity::PublicKey {
+        self.request.reply_to.identity()
```

Comment:
while it's fine now, i feel it will be a problem in the long-term when this field goes away due to our favourite SURBs